### PR TITLE
Configurable 2 fingers pinch gesture

### PR DESCRIPTION
**Pull request by osleg**
- Added pinch gestures
Basic implementation enables one shot pinch gesture.
That is the command from config file will run only once as soon as
fingers move 50% from the initial fingers position.

- Added a configurable distance variable to pinch gesture.
I felt that pinch gesture is kinda tricky to execute on touchpad,
so I changed a way of calculation when to trigger the command and
added the distance for fingers travel to configuration.
Basically a `0.5` distance feels quite ok to me, but it becomes really
snappy at `0.1`.

### CODE REVIEW

**Comment by osleg** [comment]
Closing in favor of #32
